plugins/techdocs/src/reader/components/TechDocsBuildLogs.tsx

```diff
@@ -83,15 +84,17 @@ export const TechDocsBuildLogsDrawerContent = ({
         </IconButton>
       </Grid>
 
-      <LazyLog
-        text={
-          buildLog.length === 0 ? 'Waiting for logs...' : buildLog.join('\n')
-        }
-        extraLines={1}
-        follow
-        selectableLines
-        enableSearch
-      />
+      <Suspense fallback={<LinearProgress />}>
```

Comment:
Should always prefer the Progress from core-components (see elsewhere in the PR too)

Reply:
Yep, tried to fix that in the places I copied it from too 😆